streamer/src/nonblocking/quic.rs

```diff
@@ -1488,17 +1498,14 @@ pub mod test {
         let s2 = conn2.open_uni().await;
         if let Ok(mut s2) = s2 {
             s1.write_all(&[0u8]).await.unwrap();
-            s1.finish().await.unwrap();
+            s1.finish().unwrap();
             // Send enough data to create more than 1 chunks.
             // The first will try to open the connection (which should fail).
             // The following chunks will enable the detection of connection failure.
             let data = vec![1u8; PACKET_DATA_SIZE * 2];
             s2.write_all(&data)
                 .await
                 .expect_err("shouldn't be able to open 2 connections");
-            s2.finish()
-                .await
-                .expect_err("shouldn't be able to open 2 connections");
```

Comment:
why remove this?

Reply:
In Quinn 0.11.x finish is made synchronous, it does no longer return error under this condition.